libcloudforensics/providers/azure/forensics.py

```diff
@@ -125,3 +125,66 @@ def CreateDiskCopy(
     raise RuntimeError(error_msg)
 
   return new_disk
+
+
+def StartAnalysisVm(
+    resource_group_name: str,
+    vm_name: str,
+    boot_disk_size: int,
+    ssh_public_key: str,
+    cpu_cores: int = 4,
+    memory_in_mb: int = 8192,
+    region: str = 'eastus',
+    attach_disks: Optional[List[str]] = None,
+    tags: Optional[Dict[str, str]] = None,
+    dst_profile: Optional[str] = None
+    ) -> Tuple['compute.AZVirtualMachine', bool]:
+  """Start a virtual machine for analysis purposes.
+
+  Look for an existing Azure virtual machine with name vm_name. If found,
+  this instance will be started and used as analysis VM. If not found, then a
+  new vm with that name will be created, started and returned. Note that if a
+  new vm is created, you should provide the ssh_public_key parameter.
+
+  Args:
+    resource_group_name (str): The resource group in which to create the
+        analysis vm.
+    vm_name (str): The name for the virtual machine.
+    boot_disk_size (int): The size of the analysis VM boot disk (in GB).
```

Comment:
We don't use this notation with other providers; `memory_in_mb` was named that way so it would match the official Azure notation (https://docs.microsoft.com/en-us/rest/api/compute/virtualmachinesizes/list). If this is bothering us I'd rather rename `memory_in_mb` to just `memory` and specify its unit in the docstring only. Wdyt?

Reply:
I agree with you and would be happy to leave it, just made sure that it was a conscious choice :) Great work Theo!